packages/content/types/database.d.ts

```diff
@@ -0,0 +1,112 @@
+import type { Loki, Collection } from '@lokidb/loki';
+import type { Queue, DefaultAddOptions } from 'p-queue';
+import type { FSWatcher } from 'chokidar'
+import type PriorityQueue from 'p-queue/dist/priority-queue';
+
+import type { IMarkdown } from './parsers/markdown';
+import type { IYAML } from './parsers/yaml';
+import type { ICSV } from './parsers/csv';
+import type { IXML } from './parsers/xml';
+import type { QueryBuilder } from './query-builder'
+
+type Parser = (file: string) => any;
+
+interface DatabaseOptions {
+  dir: string;
+  cwd: string;
+  markdown: any;
+  yaml: any;
```

Comment:
markdown, yaml, csv, xml can be typed (see options for @nuxt/content)

Reply:
I added all options to the database.d.ts
Maybe we want to export all these types aswell?

Reply:
Done